app/editor/menus/tableCol.tsx

```diff
@@ -156,6 +166,19 @@ export default function tableColMenuItems(
           label: dictionary.deleteColumn,
           icon: <TrashIcon />,
         },
+        {
+          name: "separator",
+        },
+        {
+          name: "spaceColumnsEvenly",
```

Comment:
UI wise this is fine for now, but as the comment on the issue i think we should (separately) change this entire toolbar to be a menu instead and then it will be much clearer.